index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+<head>    
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="start button">Let's Start</div>
+    </div>
+    <script src="js/game.js"></script>
```

Comment:
For a simple game like this, I think there's no need to split your code up into 4 different files. Might make it harder to debug, not to mention read.

You will learn refactoring later on.